### DISCUSSION TITLE
Configurable Models

### DISCUSSION THREAD

**Discussion by Arne-Jan** [ash-jc-allen/short-url#138] [Ideas]
Hello, i would like to use this package in my project, which has a tenant structure ([tenancy for laravel](https://tenancyforlaravel.com/)). 

The tenant structure i'm using changes the database connection from all the models to the tenant databases.
The problem i currently have is that i can't change the connection on the ShortURL and ShortURLVisit models to the central table.

I also want to add some additional fields to the ShortURL model, which would be possible if the model can be configurable.

Would you be willing to merge a PR which contains configurable models? 
Where the path for both ShortURL and ShortURLVisit model are defined in the config.

**Comment by ash-jc-allen** [comment]
Hey!

There's an open PR at the moment that I think might add some of the functionality that you need: https://github.com/ash-jc-allen/short-url/pull/136

Do you think the changes in this PR would work for your project? 🙂

I would potentially consider a PR to allow extra data to be stored on the `ShortURL` model too. It's something that I was thinking about adding in the future myself, so I definitely agree with the general idea 😄
